src/components/organisms/TestConfigurationForm/Custom/SourceEditFormFields.tsx

```diff
@@ -1,13 +1,10 @@
 import {Path, Revision} from '@molecules';
 
 import {StyledFormSpace} from '../TestConfigurationForm.styled';
+import {Props} from '../utils';
 
-type CustomFormFieldsProps = {
-  executorType: string;
-};
-
-const CustomFormFields: React.FC<CustomFormFieldsProps> = props => {
-  const {executorType} = props;
+const CustomFormFields: React.FC<Partial<Props>> = props => {
+  const {executorType} = props as Pick<Props, 'executorType'>;
```

Comment:
It should anyway detect it correctly:
```suggestion
  const {executorType} = props;
```